### ISSUE TITLE
Audio Mixer: No focus styles on buttons, switches, ansd sliders

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#204]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/audiomixer/

Only the text inputs have focus styles so it is impossible to know where one is. They do work correctly with the keyboard if you guess which control you are on.
